analyzers/tests/SonarAnalyzer.UnitTest/TestCases/StringLiteralShouldNotBeDuplicated.cs

```diff
@@ -5,37 +5,37 @@ public class Program
         private string empty = "";
 
         public const string NameConst = "foobar"; // Noncompliant {{Define a constant instead of using this literal 'foobar' 8 times.}}
-//                                      ^^^^^^^^
+        //                              ^^^^^^^^
```

Comment:
Let's add a case like in the failed ITs with double quotes inside and asserted message, to see the message here for that scenario.

Reply:
I also added a test for VB.